### ISSUE TITLE
Line parsing flexibility

### ISSUE THREAD

**Issue by vreuter** [pepkit/geofetch#22]
We should be able to overcome some SOFT file problems; for now we can skip over such lines but should consider populating, e.g., missing `sample_name` column in such cases, if that's an/the underlying issue.

> From a collaborator:
> 
> I followed a tip and run geofetch under 2.7 - it returns all files, great :)
>
> Unfortunately, when I tried run it exactly with GSE12417 I received:

```
  Found sample GSM311598Traceback (most recent call last):

  File "./Library/Frameworks/Python.framework/Versions/2.7/bin/geofetch", line 11, in <module>

    sys.exit(main())

  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/geofetch/geofetch.py", line 776, in main

    run_geofetch(sys.argv[1:])

  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/geofetch/geofetch.py", line 471, in run_geofetch

    pl = parse_SOFT_line(line)

  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/geofetch/geofetch.py", line 169, in parse_SOFT_line

    return {elems[0].rstrip(): elems[1].lstrip()}

IndexError: list index out of range
```
> And script was not able to produce annotation file. 
> 
> I will be grateful for any idea!

**Comment by vreuter** [comment]
Reported by collaborator via @j-lawson

**Comment by Khoroshevskyi** [comment]
@nsheff 

1) I think the code for the "raw data", esspecially for PEP creation has to be changed a bit. The problem of sample_name in the "raw data" will take some time to fix it, so I would skiped that part in the v0.9.0 release.

2) For a processed data sample_name column already uses Sample_title column. So this issue is outdated

**Comment by nsheff** [comment]
It should be possible to download metadata for accessions that lack raw data.
